input/P15tSZ.yml

```diff
@@ -0,0 +1,12 @@
+mapper_class: MapperP15tSZ
+map_name: P15tSZ
+file_map: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-milca-ymaps_2048_R2.00.fits
+file_noise: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-milca-stddev_2048_R2.00.fits
+file_mask: /mnt/extraspace/damonge/Datasets/P15tSZ/COM_CompMap_Compton-SZMap-masks_2048_R2.01.fits
```

Comment:
These paths are wrong. Should be `/mnt/extraspace/damonge/Datasets/Planck_tSZ/COM_CompMap_YSZ_R2.02/`

Reply:
Also, shouldn't there be a `ps_mask` and a `gp_mask`?

Reply:
you are right but they are the same file

Reply:
Also updated the paths.